wazuhapp/opensearch-dashboards/Docker/build.sh

```diff
@@ -66,11 +71,11 @@ prepare_env() {
         exit 1
     fi
 
-    plugin_platform_node_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
-                          print(pkg["engines"]["node"])')
+    plugin_platform_node_version=$(python -c 'import json, os; f=open("/tmp/.nvmrc"); pkg=f.readline(); f.close();\
```

Comment:
**suggestion**: maybe a simple `cat` command is enough for this case.

Reply:
Solved in [b8ffbce](https://github.com/wazuh/wazuh-packages/pull/2395/commits/b8ffbcea43a36bf146a5bad907a622f3c422238b)